### DISCUSSION TITLE
GitHub commit deployments not triggering anymore

### DISCUSSION THREAD

**Discussion by nadecancode** [vercel/vercel#6124] [Help]
![image](https://user-images.githubusercontent.com/20222292/115172798-da120380-a07a-11eb-8a67-3e8b2abd8a18.png)
Like this, it was working until today. No configuration changes. Does anyone know if Vercel is having downtime for deployments or it's due to some unprecedented changes?

**Comment by chinchang** [comment]
Same issue here. Nothing is triggering deployments at all! This is Critical! ⚠️

**Comment by paulogdm** [comment]
Hello,

After an investigation on our side, we confirmed it was a GitHub outage. Unfortunately, nothing appears on https://www.githubstatus.com/history as of now. Since they recovered, everything should work as expected.
